WooCommerce/Classes/ViewRelated/Orders/OrdersRootViewController.swift

```diff
@@ -114,37 +159,41 @@ private extension OrdersRootViewController {
             return isSimplePaymentsExperimentalToggleEnabled && isInPersonPaymentsConfigured
         }()
         let buttons: [UIBarButtonItem?] = [
-            ordersViewController.createSearchBarButtonItem(),
-            shouldShowSimplePaymentsButton ? ordersViewController.createAddSimplePaymentsOrderItem() : nil
+            createSearchBarButtonItem(),
+            shouldShowSimplePaymentsButton ? createAddSimplePaymentsOrderItem() : nil
         ]
         navigationItem.rightBarButtonItems = buttons.compactMap { $0 }
     }
 
-    func configureContainerView() {
+    func configureFiltersBar() {
+        // Display the filtered orders bar
+        // if the feature flag is enabled
+        let isOrderListFiltersEnabled = ServiceLocator.featureFlagService.isFeatureFlagEnabled(.orderListFilters)
+        if isOrderListFiltersEnabled {
+            stackView.addArrangedSubview(filtersBar)
+        }
+        filtersBar.onAction = { [weak self] in
+            self?.filterButtonTapped()
+        }
+    }
+
+    func configureChildViewController() {
+        // Configure large title using the `hiddenScrollView` trick.
         if ServiceLocator.featureFlagService.isFeatureFlagEnabled(.largeTitles) {
             hiddenScrollView.configureForLargeTitleWorkaround()
             // Adds the "hidden" scroll view to the root of the UIViewController for large title workaround.
             view.addSubview(hiddenScrollView)
+            view.sendSubviewToBack(hiddenScrollView)
             hiddenScrollView.translatesAutoresizingMaskIntoConstraints = false
             view.pinSubviewToAllEdges(hiddenScrollView, insets: .zero)
```

Comment:
It is still needed since the OrderListViewController is used by both `OrdersRootViewController` (the view controller presented by the tab) and by the order search. This means that the Filters Bar is put inside the `OrdersRootViewController` and it's indipendent from the Order List.